docs/dev_tools/third_party.md

```diff
@@ -77,6 +77,34 @@ We will do our best to maintain the exported libraries and project files, but pl
 
 ## Third party tool notes
 
+### Makefiles and mbed OS 2 projects
+
+When you export an mbed OS 2 project with a makefile exporter or an exporter that uses a makefile, you may have to modify the flags to remove some linker flags. You may see an error, like the one below, in your compiler output.
```

Comment:
- Please capitalize "Makefile"
- Please change "like" to "such as" because "like" implies similar to but not including, and "such as" implies similar to and including